src/dex/maker-psm/maker-psm-integration.test.ts

```diff
@@ -14,13 +15,9 @@ const TokenA = Tokens[network][TokenASymbol];
 const TokenBSymbol = 'DAI';
 const TokenB = Tokens[network][TokenBSymbol];
 
-const tokenAAmounts = [BigInt('0'), BigInt('100000000'), BigInt('200000000')];
+const tokenAAmounts = [BI_0, BI_POWS[8], BigInt('200000000')];
```

Comment:
Additional constants potential `BigInt('200000000')` and the next line `BigInt('2000000000000000000')`

Reply:
Its in tests. But I replaced them too